examples/aggregator/config.toml

```diff
@@ -1 +1,3 @@
 grpc_server_listen_address = "[::]:8080"
+grpc_client_connect_address = "https://localhost:8888"
+aggregator_module_hash = "05493131b2041ee7fbbcb41758133b420862c57374d318454646f51014afa838"
```

Comment:
I guess this was the only way of avoiding the infinite regress of the hash including itself as part of the hash calculation? :) Apart from splitting the aggregator into a separate wasm module (which BTW I guess would be the correct way of doing this, since currently this hash includes all the nodes, including the public ones, so if you change a single string in the router node, for instance, this hash will also change).

Reply:
I think we had this pattern in TIR (separate modules for nodes) but then decided to merge them together.
Not sure that we need separate modules in this example, since this example is very simple right now.

Reply:
And also it would require to make separate modules in all examples, where we want to check incoming modules

Reply:
It would only be necessary for nodes that perform declassification based on wasm hash or signatures. For others, there is no point in splitting them.

Reply:
Should we add a function that returns hash and public key of a module, for these purposes?

Reply:
And we probably wouldn't need to split modules (not sure yet)

Reply:
Will divide modules as a separate PR